aptos-move/parallel-executor/src/proptest_types/bencher.rs

```diff
@@ -36,6 +36,7 @@ impl<K, V> Bencher<K, V>
 where
     K: Hash + Clone + Debug + Eq + Send + Sync + PartialOrd + Ord + Arbitrary + 'static,
     V: Clone + Eq + Send + Sync + Arbitrary + 'static,
+    Vec<u8>: From<V>,
```

Comment:
this is very weird pattern, I thought it should be V: Into<Vec<u8>>

Reply:
cargo suggested doing this, the pattern or From is somehow preferred to Into or smt. Anyway, I think I'm gonna have to change to AsRef implementation instead so it will take care of this.

Reply:
I think if you implement Into you get From for free